cdisc_rules_engine/dataset_builders/variables_metadata_dataset_builder.py

```diff
@@ -12,6 +12,7 @@ def build(self):
         variable_size
         variable_data_type
         variable_format
+        define_variable_mandatory
```

Comment:
This should go in `variables_metadata_with_define_dataset_builder`, not here

Reply:
Moved it to variables_metadata_with_define_dataset_builder